recipes/SOMOS/ttseval/hparams/train_ssl_hubert_xformer.yaml

```diff
@@ -0,0 +1,92 @@
+# ############################################################################
+# Model: SSL with HuBERT (training from scratch)
+# Authors:  Artem Ploujnikov
+# # ############################################################################
+
+
+# Seed needs to be set at top of yaml, before objects with parameters are instantiated
+seed: 42
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+
+data_folder: !PLACEHOLDER
+output_folder: !ref results/ssl_hubert_xformer/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+train_regression_metric: False
+batch_size: 4
+num_workers: 4
+use_transcripts: False
+src_sample_rate: 24000
+tgt_sample_rate: 16000
+contrastive: False
+contrastive_loss_weight: 0.8
+contrastive_min_delta: 0.5
+
+
+lr: 0.0001
+dataloader_options:
+    batch_size: !ref <batch_size>
+    num_workers: !ref <num_workers>
+
+number_of_epochs: 50
+ckpt_interval_minutes: 15
+
+hubert_source: facebook/hubert-base-ls960
+hubert_freeze_feature_extractor: True
+hubert_freeze: False
+
+
+splits: ["train", "valid", "test"]
+subset: "full"
+skip_prep: False
+
+train_annotation: !ref <data_folder>/train.csv
+valid_annotation: !ref <data_folder>/valid.csv
+test_annotation: !ref <data_folder>/test.csv
+
+d_model: 512
+d_ffn: 2048
+num_layers: 4
```

Comment:
we may need to update the best config later for example wavlm large + 3-layer encoder, as well as the lr and the dropout, etc.